src/components/IncidentList.vue

```diff
@@ -47,10 +46,17 @@ export default {
     })
 
     const filteredList = computed(() =>
-      list.value.filter((i) => props.city === '' || i.city === props.city),
+      list.value.filter(
+        (i) => (props.city === '' || i.city === props.city) && i.id !== props.excludeId,
+      ),
     )
 
-    return { filteredList, selectedCityName, pluralize }
+    return { filteredList, selectedCityName }
+  },
+  methods: {
+    pluralize(n, singular, plural) {
+      return `${n} ${n === 1 ? singular : plural}`
+    },
```

Comment:
nit: `pluralize` isn't a method, it's a pure function, because it depends only on its parameters (i.e. it doesn't access anything from `this`)

Reply:
fixed this